### PR TITLE
Make the label configurable

### PR DESCRIPTION
**Pull request by karellm**
Hey,

First thanks for the neat addon :) This PR is for the rest of the world that does not speak english. I need to change the text to different languages.

### CODE REVIEW

**Comment by alechirsch** [comment]
Awesome, is nice to see that it is being used. This looks good. Can you add to the documentation for this as well?

**Comment by karellm** [comment]
@alechirsch Not only it is used but I got praised for it by our users 😄  

I updated the README (changes are amended to keep the git history clean).

**Comment by alechirsch** [comment]
I was thinking more about this, can we call the variable 'searchLabel'? 'label' seems too generic, I have another label in there for suggested filters that we may want to make configurable as well.

**Comment by alechirsch** [comment]
@karellm I will merge this after the variable name change

**Comment by karellm** [comment]
@alechirsch Agreed (I actually debated that one). Fixed!

File: README.md [hunk 1]
```diff
@@ -9,9 +9,18 @@ See the demo at [https://alechirsch.github.io/ember-search-flow/](https://alechi
...
+	parameters=parameters
+	query=query
+	onQueryUpdated=(action 'onQueryUpdated')
+	onValueUpdated=(action 'onValueUpdated')
+}}
 ```
 
+Options | Default Value | Description
+---|---|---
+`label` | `'Add filters'` | The text displayed next to the button to add filters
```
Comment (alechirsch) [inline README.md]: @karellm You missed this one

Comment (karellm) [reply README.md]: Done!

**Comment by alechirsch** [comment]
@karellm Thanks for this. Please let me know about any other feedback you might have. I will publish this soon

**Comment by alechirsch** [comment]
Released on 0.4.0

**Pull request merged**